### PR TITLE
add option to print version. Fixes #358

### LINKED ISSUE #358: Code Deploy - missing settings
I'm trying to define my "LoadBalancerInfo" using "AWS::CodeDeploy::DeploymentGroup":.  I get an error that it is an undefined method.  I tried updating your types.yaml file, but I still get an undefined method error.  I noticed from the AWS documentation that there are a few settings missing from CodeDeploy::DeploymentGroup...

```
"AlarmConfiguration" : AlarmConfiguration,
"AutoRollbackConfiguration" : AutoRollbackConfiguration,
"DeploymentStyle" : DeploymentStyle,
"LoadBalancerInfo" : LoadBalancerInfo,
"TriggerConfigurations" : [ TriggerConfig, ... ]
```

AWS docs - http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codedeploy-deploymentgroup.html#cfn-codedeploy-deploymentgroup-loadbalancerinfo

I'm not sure what I need to modify beyond the types.yaml, but I can fix and provide a pull request if you point me in the right direction.  Thanks!

### PR DESCRIPTION
**Pull request by gergnz**

### CODE REVIEW

**Comment by kornypoet** [comment]
You will need to `exit()` there in order to behave like the rest of the cli. Can you also add a quick Aruba test?

**Comment by gergnz** [comment]
> You will need to exit() there
Sorry where will I need to `exit()` @kornypoet ?

File: bin/cfndsl [hunk 1]
```diff
@@ -95,13 +99,18 @@ if options[:update_spec]
   STDERR.puts "Specification successfully written to #{CfnDsl.specification_file}"
 end
 
+if options[:assetversion]
+  spec_file = JSON.parse File.read(CfnDsl.specification_file)
+  STDERR.puts spec_file['ResourceSpecificationVersion']
+end
```
Comment (kornypoet) [inline bin/cfndsl]: I was thinking here. But I see you included similar logic on line 113 instead. That works even better.

**Comment by kornypoet** [comment]
👍

**Pull request merged**